src/test/java/core/ModelTest.groovy

```diff
@@ -0,0 +1,119 @@
+package core
+import application.fxobjects.graph.cell.Cell
+import application.fxobjects.graph.cell.RectangleCell
+
+import static org.mockito.Mockito.mock
+/**
+ * Created by Ties on 11-5-2016.
+ */
+class ModelTest extends GroovyTestCase {
+
+    void testConstructor() {
+        Model m = new Model()
+        assertNotNull(m)
+    }
+
+    void testClear() {
```

Comment:
You should first add some dummy data before calling Model.clear() as the data structures do not contain any data before the clearing.